convert/convert.go

```diff
@@ -97,6 +109,12 @@ func Convert(fsys fs.FS, ws io.WriteSeeker) error {
 		return err
 	}
 
+	if t, ok := conv.(moreParser); ok {
```

Comment:
what is "more" here?

Reply:
`more` is more configurations. This can be anything other than the configurations already parsed. In the case of BERT, this is the pooling configuration which is possibly parsed from `modules.json` and `pooling/config.json`